scripts/tempesta.sh

```diff
@@ -258,6 +299,29 @@ start_tempesta_and_check_state()
 	fi
 }
 
+start_tfw_logger()
+{
+	if [ $tfw_logger_should_start -eq 0 ]; then
+		return
+	fi
+
+	if [ -z "$mmap_host" ] || [ -z "$mmap_log" ]; then
+		error "You need to specify 'mmap_host' and 'mmap_log' "`
+		      `"if access_log mmap was specified"
+		return
+	fi
+
+	utils/tfw_logger -H "$mmap_host" -l "$mmap_log" -u "$mmap_user" -p "$mmap_password" ||
```

Comment:
I don't see log file in my system. Can you check please

Reply:
And I don't see any daemon in `ps -A` output

Reply:
We need to check that daemon was started not only code return code.

Reply:
Done.